feat(call-pill): add default toggled state (#1261)
dialpad/dialtone-vue: recipes/conversation_view/feed_pill/feed_item_pill.mdx

## Variants

> Can enable the initial toggled state

Reviewer: ```suggestion
Set the initial toggled state
```

<Canvas>
  <Story of={FeedItemPillStories.DefaultToggledVariant} />
</Canvas>

> Can enable the initial toggled state, and prevent to be toggled

Reviewer: ```suggestion
Set the initial toggled state, and prevents toggle.
```

  <Story of={FeedItemPillStories.NoToggleVariant} />
</Canvas>

## Base Style

<Canvas>
  <Story of={FeedItemPillStories.Default} />
</Canvas>

> 

Reviewer: Base style should always be the first example

Author: I feel like the control should be next before the variants? otherwise it feels odd to be "controlling" the props and having to scroll to see the changes in the base style.

Reviewer: You're not supposed to use controls to see changes on documentation, that's what the playground is for. Having the props there is so you can read abouth props and what they're meant for.
